client/src/AddVideo.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+export default function AddVideo({ addVideo }) {
+  let [input, setInput] = useState({
+    title: "",
+    url: "",
+    rating: 0,
+  });
+
+  const handleChange = (e) => {
+    setInput({ ...input, [e.target.name]: e.target.value });
```

Comment:
Looks good, just a minor suggestion, would it be possible to destructure `name` and `value` from e.target as variables?